community/ucpp/APKBUILD

```diff
@@ -1,30 +1,27 @@
 # Contributor: Timo Teräs <timo.teras@iki.fi>
 # Maintainer: Timo Teräs <timo.teras@iki.fi>
 pkgname=ucpp
-pkgver=1.3.2
+pkgver=1.3.5
 pkgrel=0
 pkgdesc="C preprocessor which is embeddable, quick, light and C99 compliant"
-url="https://code.google.com/p/ucpp/"
+options="!check" # No testsuite
+url="https://gitlab.com/scarabeusiv/ucpp"
```

Comment:
This seems to be a fork. Is the other upstream dead? Or is there some specific reasons to switch to this fork?

Reply:
that upstream had no new releases since 2011.